### PR TITLE
Entity class

### PR DESCRIPTION
**Pull request by NoePolytech**

### CODE REVIEW

File: Engine/Entity/Entity.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
+using System.Data;
+using System.Numerics;
```
Comment (jumsee) [inline Engine/Entity/Entity.cs:2]: These are useless right?

Comment (NoePolytech) [reply Engine/Entity/Entity.cs]: Oh.... embarassing

Comment (jumsee) [reply Engine/Entity/Entity.cs]: embarrassing*

**Review by jumsee — APPROVED** [review]
Seems good to me

**Pull request merged**